### PR TITLE
mean and std values are not correct

### PR DESCRIPTION
**Pull request by erjel**
Just out of curiosity I checked the mean and std values in `exercise_cifar_Keras`
```
print(np.mean(X_train, axis=(0, 1, 2)))
print(np.std(X_train, axis=(0, 1, 2)))
```
Which resulted in:
`mean= [125.30691805 122.95039414 113.86538318]`
`std=[62.99321928 62.08870764 66.70489964]`

This is far away from the suggested:
`mean = [0.4914, 0.4822, 0.4465]`
`std = [0.2470, 0.2435, 0.2616]`

When trained with the 'old' normalization values on the network
```
Layer (type)                 Output Shape              Param #   
=================================================================
conv2d_1 (Conv2D)            (None, 30, 30, 32)        896       
_________________________________________________________________
max_pooling2d_1 (MaxPooling2 (None, 15, 15, 32)        0         
_________________________________________________________________
conv2d_2 (Conv2D)            (None, 13, 13, 64)        18496     
_________________________________________________________________
max_pooling2d_2 (MaxPooling2 (None, 6, 6, 64)          0         
_________________________________________________________________
conv2d_3 (Conv2D)            (None, 4, 4, 128)         73856     
_________________________________________________________________
flatten_1 (Flatten)          (None, 2048)              0         
_________________________________________________________________
dense_1 (Dense)              (None, 512)               1049088   
_________________________________________________________________
activation_1 (Activation)    (None, 512)               0         
_________________________________________________________________
dropout_1 (Dropout)          (None, 512)               0         
_________________________________________________________________
dense_2 (Dense)              (None, 10)                5130      
_________________________________________________________________
activation_2 (Activation)    (None, 10)                0         
=================================================================
```

I got an maximal accuracy on the training set of 85%/ 62%  (training/ validation).
On the new values the network converged to an accuracy of 97%/ 66% (training/ validation).

By doing so I also mitigated an increasing loss function for large epochs:
Screenshot from 2019-12-15 10-55-46

So I change the suggestion in the notebook.

### CODE REVIEW

**Comment by vzinche** [comment]
Thanks for noticing! The thing is that the original images are in [0,255] range. And while pytorch brings them to [0,1] in ToTensor transform, Keras doesn't. So just for the sake of consistency might be cleaner to divide the image by 255, and then to normalize to the originally suggested mean and std. @pejmanrasti could you take a look?

**Comment by pejmanrasti** [comment]
Hi, 
As Valentyna mentioned, pixel values by themselves are between 0 to 255. We should first map them between 0 and 1 then do the next steps. I added it to the description of the cell as well to avoid misunderstandings.
